packages/flutter_tools/test/general.shard/xcode_backend_test.dart

```diff
@@ -755,261 +762,1021 @@ void main() {
+              '${flutterRootDir.path}/bin/cache/artifacts/engine/ios/Flutter.xcframework/ios-arm64/Flutter.framework/Info.plist',
+            )
+            ..createSync(recursive: true)
+            ..writeAsStringSync(matchingInfoPlist);
+
+          const flutterBuildDir = 'build';
+          final Directory nativeAssetsDir = memoryFileSystem.directory(
+            '/path/to/my_flutter_app/$flutterBuildDir/native_assets/ios/',
+          );
+          nativeAssetsDir.createSync(recursive: true);
+          const ffiPackageName = 'package_a';
+          final Directory ffiPackageDir = nativeAssetsDir.childDirectory(
+            '$ffiPackageName.framework',
+          )..createSync();
+          nativeAssetsDir.childFile('random.txt').createSync();
+          const infoPlistPath = 'Runner.app/Info.plist';
+          final File infoPlist = memoryFileSystem.file('${buildDir.path}/$infoPlistPath');
+          infoPlist.createSync(recursive: true);
+          const buildMode = 'Debug';
+          final testContext = TestContext(
+            <String>['embed_and_thin', 'ios'],
+            <String, String>{
+              'BUILT_PRODUCTS_DIR': buildDir.path,
+              'CONFIGURATION': buildMode,
+              'FLUTTER_ROOT': flutterRootDir.path,
+              'INFOPLIST_PATH': infoPlistPath,
+              'SOURCE_ROOT': platformDirPath,
+              'FLUTTER_APPLICATION_PATH': appPath,
+              'FLUTTER_BUILD_DIR': flutterBuildDir,
+              'TARGET_BUILD_DIR': targetBuildDir.path,
+              'FRAMEWORKS_FOLDER_PATH': frameworksFolderPath,
+              'EXPANDED_CODE_SIGN_IDENTITY': '12312313',
+              'FLUTTER_FRAMEWORK_SWIFT_PACKAGE_PATH': flutterSwiftPackageDir.path,
+              'SDKROOT': 'iphoneos',
+            },
+            commands: <FakeCommand>[
+              FakeCommand(
+                command: <String>[
+                  'mkdir',
+                  '-p',
+                  '--',
+                  targetBuildDir.childDirectory(frameworksFolderPath).path,
+                ],
+              ),
+              FakeCommand(
+                command: <String>[
+                  'rsync',
+                  '-8',
+                  '-av',
+                  '--delete',
+                  '--filter',
+                  '- .DS_Store',
+                  buildDir.childDirectory('App.framework').path,
+                  targetBuildDir.childDirectory(frameworksFolderPath).path,
+                ],
+              ),
+              // FakeCommand(
+              //   command: <String>[
+              //     'rsync',
+              //     '-8',
+              //     '-av',
+              //     '--delete',
+              //     '--filter',
+              //     '- .DS_Store',
+              //     buildDir.childDirectory('Flutter.framework').path,
+              //     '${targetBuildDir.childDirectory(frameworksFolderPath).path}/',
+              //   ],
+              // ),
```

Comment:
Was this intended to be removed? It's not clear why it's here but commented out.